### DISCUSSION TITLE
11.0.6

### DISCUSSION THREAD

**Discussion by maxkatz6** [AvaloniaUI/Avalonia#13841] [General]
https://www.nuget.org/packages/Avalonia/11.0.6

**Full Changelog**: https://github.com/AvaloniaUI/Avalonia/compare/11.0.5...11.0.6

<hr /><em>This discussion was created from the release <a href='https://github.com/AvaloniaUI/Avalonia/releases/tag/11.0.6'>11.0.6</a>.</em>

**Comment by fforjan** [comment]
Does this fully support net8.0 ? or is there pending issues with net8.0 ? I could not clearly find this information

**Reply by maxkatz6** [reply]
By default, you can expect any Avalonia version to support newer .NET SDK versions. That's because .NET ecosystem is normally backward compatible.

Exceptions can happen though, like with .NET 8 WASM support which introduced breaking changes - these were fixed in 11.0.6.
